bin/index.js

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+'use strict';
+
+const example = require('../lib/example');
```

Comment:
`bin/index.js` doesn't seem necessary for this diagnostic, but I do see it in `node-template@016/master`.